### PR TITLE
feat(fa): Municipality overview

### PR DESCRIPTION
**Pull request by Rikkiabb**
# ...
https://app.asana.com/0/1201301944561348/1200757562424376

## What
Super admin can fetch list of all municipalities

## Why
Needed for super admin system

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/financial-aid/web-veita/src/routes/Municipalities/municipalities.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        <Text as="h1" variant="h1">
+          Notendur
+        </Text>
+        <Button size="small" icon="add" variant="ghost">
+          Nýtt sveitarfélag
+        </Button>
+      </Box>
+
+      {municipalities && (
+        <div className={`${styles.wrapper} hideScrollBar`}>
```
Comment (MargretFinnboga) [inline apps/financial-aid/web-veita/src/routes/Municipalities/municipalities.tsx:57]: Are we making sure the table only takes grid columns five here ? 😁

**Review by MargretFinnboga — COMMENTED** [review]
Mr world wide, check it 🌍

File: apps/financial-aid/web-veita/src/routes/Municipalities/municipalities.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      isLoading={loading}
+      loader={<ApplicationOverviewSkeleton />}
+    >
+      <Box
+        className={`${styles.header} contentUp delay-25`}
+        marginTop={15}
+        marginBottom={[2, 2, 4]}
+      >
+        <Text as="h1" variant="h1">
+          Notendur
```
[the remark this answers is not in the document]
Comment (Rikkiabb) [reply apps/financial-aid/web-veita/src/routes/Municipalities/municipalities.tsx:49]: 🙈

File: apps/financial-aid/web-veita/src/components/SuperAdminSideNavItems/SuperAdminSideNavItems.css.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+export const sideNavBarButton = style({
+  display: 'flex',
+  alignItems: 'center',
+  marginBottom: '8px',
+})
+
+export const sideNavBarButtonIcon = style({
+  marginRight: theme.spacing[2],
+  transition: 'transform 250ms ease',
+})
```
[the remark this answers is not in the document]
Comment (MargretFinnboga) [reply apps/financial-aid/web-veita/src/components/SuperAdminSideNavItems/SuperAdminSideNavItems.css.ts:13]: Ignoring this comment 🙃 ?

**Review by MargretFinnboga — APPROVED** [review]
🤩  ✡️

**Pull request merged**